module/solutions/helper.js

```diff
@@ -14,6 +13,7 @@ const surveyService = require(ROOT_PATH + '/generics/services/survey');
 const improvementProjectService = require(ROOT_PATH + '/generics/services/improvement-project');
 const appsPortalBaseUrl = process.env.APP_PORTAL_BASE_URL + "/" ;
 const userExtensionsHelperV2 = require(MODULES_BASE_PATH + "/user-extension/helperv2");
+const userService = require(ROOT_PATH + "/generics/services/users");
```

Comment:
@VISHNUDAS-tunerlabs i think we should change the entities helper functions only, making sure existing argument and response is not modified only the logic inside it is modified to work with location search and org search, we can add new arguments to that function if needed. By doing so no other files have to be modified. This is for all 3 services.

Reply:
@aks30 decided to test and go with current code